plugins/main/public/components/common/wazuh-discover/discover.scss

```diff
@@ -3,6 +3,11 @@
     height: max-content !important;
   }
 
+  .euiDataGridRowCell:focus {
+    border: 1px solid rgba(0, 107, 180, 0.4);
```

Comment:
**nitpick**: this defines a hardcoded color without managing the theme or light/dark mode.

Reply:
The style was removed in a later commit.